gui/README.md

```diff
@@ -8,30 +8,41 @@ npm install
 ### To run the GUI with Cassy
```

Comment:
Can you update the top title to `Cassy GUI` ?
(sorry I can't comment on non-changed parts)